src/app/public/modules/text-expand-repeater/text-expand-repeater.component.ts

```diff
@@ -28,12 +31,18 @@ let nextId = 0;
   ]
 })
 export class SkyTextExpandRepeaterComponent implements AfterViewInit {
+  @ContentChildren(TemplateRef)
+  private templates: QueryList<TemplateRef<any>>;
   @Input()
   public maxItems: number;
   @Input()
   public set data(value: Array<any>) {
     this.setup(value);
   }
+  /* tslint:disable:no-input-rename */
+  @Input('template')
+  public templateInput: TemplateRef<any>;
```

Comment:
Some of our other components use `TemplateRef` as an `Input`. For consistency's sake, I'd prefer to enforce the use of an `Input` named `itemTemplate` (and remove the fallback `@ContentChildren() templates`, above). Does that work for you?

(Your usage of the `#defaultItemTemplate` in the HTML will prevent this change from being a _breaking_ change; thanks for that!)

```
@Input()
public itemTemplate: TemplateRef<any>;
```